src/antidote_app.erl

```diff
@@ -25,6 +25,11 @@
 %% List of the contributors to the development of Antidote: see AUTHORS file.
 %% Description and complete License: see LICENSE file.
 %% -------------------------------------------------------------------
+
+%% @doc Starts the antidote supervisor process,
```

Comment:
I suggest adding as information that this module implements the application callbacks.

Reply:
would you like that to be added along with the current comment or do you want it to replace it? @bieniusa

Reply:
Adding :) Thanks!